### ISSUE TITLE
FeatureRequest: Mono support

### ISSUE THREAD

**Issue by awb99** [icsharpcode/SharpDevelop#706]
I am an active user of SharpDevelop -  on Windows. SharpDevelop is the best IDE for c# in my view. Compared to Visual Studio it is really slick and fast, nothing is bloated. Compared to MonoDevelop it is a complete different league; MonoDevelop ist way behind.

I have seen that the last discussion of the Linux Support mainly happened in 2008, and i think this should be discussed again in the light of new circumstances: The Linux Desktop has improved dramatically since then; Microsoft open sourced .net in big parts. WinForms support in Mono
is good now.

I would assume that the only reason for not having Linux support in SharpDevelop are a few
non safe calls, which I assume can be safely refactored.

Thanks!

**Comment by jogibear9988** [comment]
I don't think this is going to happen! Many parts of the SD UI are using WPF instead of Winforms, wich is not supported by Mono!

**Comment by linquize** [comment]
In the old days, SharpDevelop used WinForms. But now most parts have been migrated to WPF.

**Comment by christophwille** [comment]
There are no plans to get into MonoDevelop territory (after all, they could easily take our MIT code where they see fit).

**Comment by awb99** [comment]
Thanks for your comments! In this case perhaps it makes sense to run SharpDevelop in an emulator like Wine... SharpDevelop is such a good app - it should be able to run it easily on any os..
